docs/uds-packages/guide.md

```diff
@@ -30,7 +30,7 @@ Before beginning the integration process, familiarize yourself with the followin
 1. [UDS Capabilities Documentation](https://uds.defenseunicorns.com/overview/uds-structure/#uds-core-capabilities): Provides information about UDS, UDS CLI, UDS Core, and UDS Bundles.
```

Comment:
Should fix the Apache 2 stuff above and would prefer to drop notion / coda from this repo for now.

Reply:
Updated with this commit: https://github.com/defenseunicorns/uds-common/pull/346/commits/a3d73c245e39a845436bd9f386998d01a9dab07c to resolve comments.